pkg/clusterregistry/options/authorization.go

```diff
@@ -0,0 +1,70 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package options
+
+import (
+	"github.com/spf13/pflag"
+
+	"k8s.io/apiserver/pkg/authorization/authorizer"
+	"k8s.io/apiserver/pkg/authorization/authorizerfactory"
+	genericapiserver "k8s.io/apiserver/pkg/server"
+)
+
+// StandaloneAuthorizerConfig configures an authorizer for the standalone
+// cluster registry.
+type StandaloneAuthorizerConfig struct {
+	AlwaysAllow bool
```

Comment:
I suppose this structure and the `StandaloneAuthorizationOptions` structure will eventually diverge that separating them makes sense.

Reply:
That was my thought as well. I considered removing this type, though, since it's not serving any particular purpose at the moment; I kept it for consistency with the other `Options` types and to support future divergence.

Reply:
sgtm